pkg/scheduler/scheduler.go

```diff
@@ -215,7 +197,9 @@ func WithPodMaxBackoffSeconds(podMaxBackoffSeconds int64) Option {
 }
 
 var defaultSchedulerOptions = schedulerOptions{
-	schedulerName: v1.DefaultSchedulerName,
+	profiles: []schedulerapi.KubeSchedulerProfile{
+		{SchedulerName: v1.DefaultSchedulerName},
```

Comment:
Add a comment clarifying that the default provider plugins is used by default

Reply:
Done